include/circt/Dialect/Arc/Passes.td

```diff
@@ -0,0 +1,31 @@
+//===- Passes.td - Arc dialect passes ----------------------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_DIALECT_ARC_PASSES_TD
+#define CIRCT_DIALECT_ARC_PASSES_TD
+
+include "mlir/Pass/PassBase.td"
+
+def ConvertToArcs : Pass<"convert-to-arcs", "mlir::ModuleOp"> {
+  let summary = "Lower logic to state transfer arcs";
+  let constructor = "circt::arc::createConvertToArcsPass()";
+  let dependentDialects = ["arc::ArcDialect"];
+}
+
+def InlineModules : Pass<"arc-inline-modules", "mlir::ModuleOp"> {
+  let summary = "Inline modules that are instantiated once";
```

Comment:
Aren't all private modules inlined eagerly instead of just the ones instantiated only once?

Reply:
Good catch! Fixed.